cmd/add.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/railwayapp/cli/entity"
+	"github.com/railwayapp/cli/ui"
+)
+
+func stringInSlice(a string, list []*string) bool {
+    for _, b := range list {
+        if *b == a {
+            return true
+        }
+    }
+    return false
+}
+
+func (h *Handler) addFromName(ctx context.Context, pluginRequest string, plugins []*string, projectId string) error {
+	reqValid := stringInSlice(pluginRequest,plugins)
+	if !reqValid {
+		return errors.New("Plugin either has been created or not valid.")
+	}
```

Comment:
Should add a seperate check for invalid plugin. Honestly, probably just better to only have the dropdown. Args may confuse people and they'll need to know exact string (e.g postgresql vs postgres)

```suggestion
	reqValid := stringInSlice(pluginRequest ,plugins)
	if !reqValid {
		return errors.New("Plugin already exists.")
	}
```